master/internal/api_workspace.go

```diff
@@ -326,6 +384,28 @@ func (a *apiServer) PatchWorkspace(
 		insertColumns = append(insertColumns, "uid", "user_", "gid", "group_")
 	}
 
+	if req.Workspace.CheckpointStorageConfig != nil {
```

Comment:
Q: is there a way to set it to null, e.g. by passing an empty string or empty struct?

Reply:
I suppose not since an empty struct will fail the validation

I think we might need field masks here

Reply:
we should check with @maxrussell on the state of fieldmasks then.